decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -93,23 +132,53 @@ const AddCommentFormWithMutation = graphql(gql`
         __typename: 'Mutation',
         addComment: {
           __typename: 'Comment',
-          id: random.uuid(),
-          createdAt: (new Date()).toString(),
+          id: uuid(),
+          createdAt: moment().format("YYYY-MM-DD HH:mm:ss z"),
```

Comment:
Agree but I feel coercing to a date should be done by the top-level component instead, and components should always be dealing with date objects (instead of coercion being made in the "dumb" components).